Source/com/drew/imaging/mp4/Mp4Reader.java

```diff
@@ -51,10 +51,14 @@ private static void processBoxes(StreamReader reader, long atomEnd, Mp4Handler<?
             while (atomEnd == -1 || reader.getPosition() < atomEnd) {
 
                 Box box = new Box(reader);
+                if (box.type.equals("meta")) {
+                    // TODO: Figure out how to support containers that are also FullBox types (extra 4 bytes for version and flag)
+                    System.out.println("THIS NEEDS TO BE CHANGED.");
+                    reader.skip(4);
+                }
```

Comment:
I am not sure the best way to do this.

We consider "meta" boxes to be a container, which they are. However, they are also technically full box implementations, so they have an extra 4 bytes. This is a hacky way to bypass this information for now and continue reading into the container.